### PR TITLE
Improve GitHub Integration configuration docs

### PR DESCRIPTION
**Pull request by DraftProducts**
- Fix bad documentation in configuration page
- Add button to auto-generate GitHub app in GitHub Integration page
- Improve auto-generated GitHub app name in GitHub SSO

Closes : https://github.com/getsentry/self-hosted/issues/1486

### CODE REVIEW

**Comment by chadwhitacre** [comment]
Thanks @DraftProducts, kicking off the Vercel draft build ...

File: src/docs/config.mdx [hunk 1]
```diff
@@ -264,9 +264,16 @@ The secret used to verify GitHub App Webhook event.
 
 Your app's private key.
```
Comment (chadwhitacre) [inline src/docs/config.mdx:265]:
```suggestion
Your app's private key, best to use [multiline](https://yaml-multiline.info/):
```

File: src/docs/integrations/github.mdx [hunk 2]
```diff
@@ -10,26 +10,34 @@ To configure the GitHub integration you'll need to create a GitHub app and obtai
   The GitHub App name must not contain any spaces.
 </Alert>
 
+<CreateGitHubAppForm url="https://github.com/organizations/:org/settings/apps/new?name=:org-Sentry-Integration&public=false&members=read&emails=read&administration=read&contents=read&issues=write&pull_requests=write&repository_hooks=write&url=:url-prefix&callback_url=:url-prefix%2Fauth%2Fsso%2F&setup_url=:url-prefix%2Fextensions%2Fgithub%2Fsetup%2F&webhook_url=:url-prefix%2Fextensions%2Fgithub%2Fwebhook%2F&events[]=push&events[]=pull_request&webhook_active=true" defaultOrg="your-organization" defaultUrlPrefix="https://your-sentry-url-prefix.com" />
```
Comment (chadwhitacre) [inline src/docs/integrations/github.mdx:13]: Whoa, really!? 🧐

Comment (chadwhitacre) [reply src/docs/integrations/github.mdx:13]: @BYK coming back to haunt us (#143). 😁

Comment (DraftProducts) [reply src/docs/integrations/github.mdx:13]: I just used the same form present in [GitHub SSO docs](https://develop.sentry.dev/self-hosted/sso/#github-auth) page.

**Review by chadwhitacre — APPROVED** [review]
Lgtm. I think it'd be better to deduplicate the documentation between the GitHub integration and SSO pages, but this PR is still an improvement as it stands.

File: src/docs/integrations/github.mdx (hunk 2 above)
Comment (BYK) [reply src/docs/integrations/github.mdx:13]: Reusability <3 me

**Pull request merged**